.gitignore

```diff
@@ -1 +1,4 @@
 coverage.out
+vendor/
+.idea/
```

Comment:
Please avoid adding content related to your choice of tools and editors to the project.

I suggest using a global $HOME/.gitignore file as an alternative.
```
git config --global core.excludesfile $HOME/.gitignore
```